docs/website/src/pages/examples/botonic-example.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable import/no-unresolved */
+// eslint-disable-next-line node/no-missing-import
+import Head from '@docusaurus/Head'
+import React from 'react'
+
+import Analytics from '../../components/analytics'
+
+const BotonicExample = ({ title, rootId, runtimeOptions, src }) => {
+  if (typeof window !== 'undefined') {
+    const removejscssfile = (filename, filetype) => {
+      const targetelement =
+        filetype == 'js' ? 'script' : filetype == 'css' ? 'link' : 'none' //determine element type to create nodelist from
+      const targetattr =
+        filetype == 'js' ? 'src' : filetype == 'css' ? 'href' : 'none' //determine corresponding attribute to test for
+      const allsuspects = document.getElementsByTagName(targetelement)
+      for (let i = allsuspects.length; i >= 0; i--) {
+        //search backwards within nodelist for matching elements to remove
+        if (
+          allsuspects[i] &&
+          allsuspects[i].getAttribute(targetattr) != null &&
+          allsuspects[i].getAttribute(targetattr).indexOf(filename) != -1
+        )
+          allsuspects[i].parentNode.removeChild(allsuspects[i]) //remove element by calling parentNode.removeChild()
+      }
+    }
+    removejscssfile('styles.css', 'css') // Dev
+    removejscssfile('styles.9c057c15.css', 'css') // Prod
```

Comment:
There's a lot happening here? Could it be encapsulated to a function with a defined scope?

Reply:
i agree, at first i've tried to extract it into another file, but had issues to run the build. I will try to take it a look again 👍

Reply:
moved, now working 👍